internal/mtail/golden/reader_test.go

```diff
@@ -106,7 +106,7 @@ var expectedMetrics = metrics.MetricSlice{
 		LabelValues: []*metrics.LabelValue{
 			{
 				Labels: []string{},
-				Value:  datum.MakeString("hi", time.Date(2018, 6, 16, 18, 04, 0, 0, time.UTC)),
+				Value:  datum.MakeString("hi", time.Date(2018, 6, 16, 18, 0o4, 0, 0, time.UTC)),
```

Comment:
This one's a bug!  Just make it a decimal 4 please.

Reply:
done